src/lib.rs

```diff
@@ -70,6 +70,7 @@ pub struct LightningNode {
     background_processor: BackgroundProcessor,
     channel_manager: Arc<ChannelManager>,
     peer_manager: Arc<PeerManager>,
+    peer_conn_handle: JoinHandle<()>,
```

Comment:
I would rename it to `p2p_connector_handle` otherwise, it seems like it is a handle to a connection and not to the connector task.

Reply:
Fixed: aa2f6c6ad837b014a771429f51d9cc5fa2db57de